ModularTegustation/tegu_items/representative/research/lcorp.dm

```diff
@@ -146,6 +146,19 @@ GLOBAL_LIST_EMPTY(lcorp_upgrades)
 	GLOB.lcorp_upgrades += "Agent Workchance"
 	..()
 
+//EGO Gift Extractor
+/datum/data/lc13research/ego_gift_extractor
+	research_name = "Repeatable: EGO Gift Extractor"
+	research_desc = "An order of an EGO Gift Extractor. "
+	cost = AVERAGE_RESEARCH_PRICE+5
+	corp = L_CORP_REP
+
+/datum/data/lc13research/ego_gift_extractor/ResearchEffect(obj/structure/representative_console/caller)
+	if(repeat_cooldown > world.time)
```

Comment:
Give a small notification here and I'll approve it.

This is going to be experimental so....

Reply:
Done.